packages/e2e.sandbox.catchall/src/runInterfacesTest2.ts

```diff
@@ -62,6 +62,7 @@ export async function runInterfacesTest2(): Promise<void> {
     >
   >(true);
 
+  // You cannot specify both $select and $includeAllBaseObjectProperties
```

Comment:
When this failed for me during dev I as confused as to why, so extra comment for future us